CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 
-## 0.2.0 - 6 December, 2024
+## 0.2.1 - 22 January, 2024
```

Comment:
This is misleading. There was no release `0.2.1` on 22 January

Reply:
Oh yes, I have changed the file to match released versions now